src/qt/sendcoinsentry.cpp

```
@@ -55,13 +57,37 @@ SendCoinsEntry::SendCoinsEntry(const PlatformStyle *_platformStyle, QWidget *par
     delete ui;
 }
 
+void SendCoinsEntry::on_MemoTextChanged(const QString &text)
+{
+    const spark::Params* params = spark::Params::get_default();
+    int maxLength = params->get_memo_bytes();
+    bool isOverLimit = text.length() > maxLength;
+
+    if (isOverLimit) {
+        ui->messageWarning->setText("Message exceeds character 256 character limit");
+        ui->messageWarning->setVisible(true);
```

cppcheck

[performance] Function parameter 'label' should be passed by const reference.

(passedByValue)